src/components/farm/FarmDataSection.tsx

```
@@ -0,0 +1,378 @@
+  const validateField = useCallback((field: keyof FarmData, value: string): string | undefined => {
+    switch (field) {
+      case 'farmName':
+        if (!value.trim()) return 'Farm name is required';
+        if (value.trim().length < 2) return 'Farm name must be at least 2 characters';
+        break;
+      case 'location':
+        if (!value.trim()) return 'Location is required';
+        break;
+      case 'size':
+        if (!value.trim()) return 'Farm size is required';
+        const sizeNum = parseFloat(value);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)